containerd/containerd.go

```diff
@@ -149,7 +149,7 @@ func (d *Driver) createContainer(containerConfig *ContainerConfig, config *TaskC
 
 	// Enable privileged mode.
 	if config.Privileged {
-		opts = append(opts, oci.WithPrivileged)
+		opts = append(opts, oci.WithPrivileged, oci.WithAllDevicesAllowed, oci.WithHostDevices)
```

Comment:
Nice! Can we also add `oci.WithNewPrivileges` to this list to keep it compatible with `nerdctl` [`implementation`](https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/run_security_linux.go#L36-L41)?

Reply:
Done.